packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -43,16 +42,18 @@ class NetworkAssetBundle extends AssetBundle {
 }
 
 abstract class CachingAssetBundle extends AssetBundle {
-  Map<String, ImageResource> _imageCache = new Map<String, ImageResource>();
-  Map<String, Future<String>> _stringCache = new Map<String, Future<String>>();
+  final Map<String, ImageResource> imageResourceCache =
+    new Map<String, ImageResource>();
```

Comment:
nit: while you're here, for consistency, change `new Map<T, R>()` to `<T, R>{}`.